test/Sentry.Tests/PlatformAbstractions/FrameworkInstallationExtensionsTests.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using Sentry.PlatformAbstractions;
+using Xunit;
+
+namespace Sentry.Tests.PlatformAbstractions
+{
+    public class FrameworkInstallationExtensionsTests
+    {
+        [Fact]
+        public void GetVersionNumber_NullFrameworkInstallation_NullVersion()
+        {
+            FrameworkInstallation frameworkInstallation = null;
+            //Arrange
+
+            //Act
+            var version = frameworkInstallation.GetVersionNumber();
+
+            //Assert
+            Assert.Null(version);
+        }
+
+        [Fact]
+        public void GetVersionNumber_NullShortVersionAndNullVersion_NullVersion()
+        {
+            var frameworkInstallation = new FrameworkInstallation();
+            //Arrange
+
+            //Act
+            var version = frameworkInstallation.GetVersionNumber();
+
+            //Assert
+            Assert.Null(version);
+        }
+
+        [Fact]
+        public void GetVersionNumber_ValidShortVersion_ShortVersion()
+        {
+            var frameworkInstallation = new FrameworkInstallation();
+            var expectedShortVersion = "v1.2.3";
+            frameworkInstallation.ShortName = expectedShortVersion;
+            //Arrange
```

Comment:
Why is Arrange below the arrange?

Consider leaving these comments out and using the empty lines to split them.
Otherwise please move the comment above

Reply:
not sure why I did that, in any case, fixed.